### PR TITLE
Update hrv_OCRFixReplaceList.xml

### PR DESCRIPTION
**Pull request by diomed**
@niksedk  please merge.

Thnx!

[also, did you take a look @ http://git.io/vm4xu]

### CODE REVIEW

**Comment by niksedk** [comment]
Ready for merge?

Also, could one of you add a line in the change log about the new hrv + new srp ocr fix replace list?

**Comment by diomed** [comment]
better tomorrow evening. I'll still make some changes.

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 1]
```diff
@@ -1185,152 +1124,265 @@
...
+    <RegEx find="([vV])idel" replaceWith="$1idje" />
+    <RegEx find="([vV])ideo(?!ka[zs]et[aeiu]|zapis|tek[aiue]|trgovin[aeiu])" replaceWith="$1idio" />
     <RegEx find="\b([vV])išlj" replaceWith="$1iš" />
-    <RegEx find="\b([vV])odk(a|e|i|u|ama)" replaceWith="$1otk$2" />
+    <RegEx find="\b([vV])odk(a|e|i|u|ama|om)" replaceWith="$1otk$2" />
     <RegEx find="([vV])ole(n|na|no|ti|nu|nog|la|le|li)" replaceWith="$1olje$2" />
     <RegEx find="\b([vV])oz(a|u|om)?\b" replaceWith="$1lak$2" />
     <RegEx find="\b([vV])rj?edi(m|š|mo|te|li)?\b" replaceWith="$1rijedi$2" />
     <RegEx find="\b([vV])rj?ed(e|an|na|no|nost|nosti|nošću)?\b" replaceWith="$1rijed$2" />
+    <RegEx find="(vV)rj?eđa" replaceWith="$1rijeđa" />
```
Comment (xylographe) [inline Dictionaries/hrv_OCRFixReplaceList.xml]:
```
<RegEx find="([vV])rj?eđa" replaceWith="$1rijeđa" />
```
(square brackets)

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 2]
```diff
@@ -1185,152 +1124,265 @@
...
-    <RegEx find="\b([vV])enčan(a|i|ica|icu|icom|icama|je|ja|ju|jem|jima)?\b" replaceWith="$1jenčan$2" />
-    <RegEx find="veštačk(a|e|i|o|u|og|om|oj|im|ima)" replaceWith="umjetn$1" />
-    <RegEx find="Veštačk(a|e|i|o|u|og|om|oj|im|ima)" replaceWith="Umjetn$1" />
+    <RegEx find="\b([vV])j?en(ac|c[aeiu]|cem)" replaceWith="$1ijen$2" />
+    <RegEx find="\b([vV])enčan(a|i|ica|icu|icom|icama|j[eau]|jem|jima)?\b" replaceWith="$1jenčan$2" />
+    <RegEx find="veštačk" replaceWith="umjetn" />
+    <RegEx find="Veštačk" replaceWith="Umjetn" />
     <RegEx find="([vV])et(ar|ra|ru|rom|rovi|rovito|rovima)" replaceWith="$1jet$2" />
-    <RegEx find="([vV])ide(la|le|li|ti)" replaceWith="$1idje$2" />
+    <RegEx find="([vV])idel" replaceWith="$1idje" />
```
Comment (xylographe) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: I think, the "l" should not be removed?
```
<RegEx find="([vV])idel" replaceWith="$1idjel" />
```
`videla` ==> `vidjela`, not `videla` ==> `vidjea`

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 3]
```diff
@@ -1185,152 +1124,265 @@
...
+    <RegEx find="([uU])spi?j?eš([an]|n[aeiou])" replaceWith="$1spješ$2" />
     <RegEx find="([uU])sredsred" replaceWith="$1sredotoč" />
     <RegEx find="([uU])sredsređen" replaceWith="$1sredotočen" />
-    <RegEx find="\b([uU])spj?ev(am|aš|a|amo|ate|aju|ale|alo|ali|aše)" replaceWith="$1spijev$2" />
+    <RegEx find="\b([uU])spj?eva" replaceWith="$1spijev" />
     <RegEx find="([uU])ćut(im|iš|i|imo|ite|e)" replaceWith="$1šut$2" />
-    <RegEx find="univerzum(a|e|i|u|om|ima)?" replaceWith="svemir$1" />
+    <RegEx find="univerzum" replaceWith="svemir$1" />
+    <RegEx find="Univerzum" replaceWith="Svemir$1" />
     <RegEx find="uskrs(a|i|u|om|ima)?" replaceWith="Uskrs$1" />
```
Comment (xylographe) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: Perhaps, 
```
<RegEx find="\buskrs" replaceWith="Uskrs" />
```
or, 
```
<RegEx find="\buskrs(a|i|u|om|ima)?\b" replaceWith="Uskrs$1" />
```
?

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 4]
```diff
@@ -1111,31 +1035,42 @@
...
     <RegEx find="([oO])rganizov" replaceWith="$1rganizir" />
+    <RegEx find="([oO])ružij[aeu]" replaceWith="$1ružj" />
     <RegEx find="([oO])seti(o|m|š|la|li|le|ti)" replaceWith="$1sjeti$2" />
     <RegEx find="([oO])setljiv(a|i|e|o|om|im|ima|oj)?" replaceWith="$1sjetljiv$2" />
     <RegEx find="([oO])sj?e[ćč]a" replaceWith="$1sjeća" />
     <RegEx find="([oO])strv(a|u|om)" replaceWith="$1tok$2" />
     <RegEx find="([oO])sveženj(a|e|u|em|ima)" replaceWith="$1svježenj$2" />
     <RegEx find="([oO])svežava(m|š|mo|te|ju|ti|li|la|lo|ju|juć|juće|juća)?" replaceWith="$1svježava$2" />
-    <RegEx find="\b([oO])ter(a|am|amo|ate|aju)" replaceWith="$1tjer$2" />
+    <RegEx find="\b([oO])tera(m|mo|te|ju)?" replaceWith="$1tjera$2" />
```
Comment (xylographe) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: I think, there might be a `\b` missing at the end of the regular expression:
```
<RegEx find="\b([oO])tera(m|mo|te|ju)?\b" replaceWith="$1tjera$2" />
```
Unless you really want to replace every "otera" and "Otera" at the start of a word:
```
<RegEx find="\b([oO])tera" replaceWith="$1tjera" />
```
(the *optional* `(m|mo|te|ju)` group serves no purpose)

**Comment by diomed** [comment]
@niksedk please *merge* before releasing 3.4.7.

**Pull request merged**